envi/cli.py

```diff
@@ -783,6 +796,18 @@ def showmem():
         showmem()
         self.setEmptyMethod(showmem)
 
+    def validate_script(self, scriptpath):
+        try:
+            with open(scriptpath, 'rb') as f:
+                contents = f.read()
+
+            cobj = compile(contents, scriptpath, 'exec')
+            return True
+        except Exception, e:
+            pass
```

Comment:
( also, "except: pass" construct will *always* give me pause, as it's *super* dangerous :) )